collector/src/test/java/com/google/cloud/trace/zipkin/ZipkinCollectorIntegrationTest.java

```diff
@@ -0,0 +1,81 @@
+package com.google.cloud.trace.zipkin;
+
+import com.google.cloud.trace.v1.sink.TraceSink;
+import com.google.devtools.cloudtrace.v1.Trace;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Primary;
+import org.springframework.core.task.SyncTaskExecutor;
+import org.springframework.test.context.junit4.SpringRunner;
+import zipkin.Codec;
+import zipkin.Span;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+import java.util.concurrent.Executor;
+
+import static com.google.cloud.trace.zipkin.StackdriverZipkinCollector.ZIPKIN_CONFIG_NAMES;
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.verifyNoMoreInteractions;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest(classes = {StackdriverZipkinCollector.class, ZipkinCollectorIntegrationTest.TestConfiguration.class},
+    webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT,
+    properties = {ZIPKIN_CONFIG_NAMES, "zipkin.storage.stackdriver.project-id=test_project"})
+public class ZipkinCollectorIntegrationTest {
+  private Random rnd = new Random();
+
+  @MockBean
+  private TraceSink traceSink;
+
+  @Autowired
+  private TestRestTemplate restTemplate;
+
+  @Test
+  public void traceSinkGetsCalled() throws InterruptedException
+  {
+    this.restTemplate.postForLocation("/api/v1/spans", createTestSpans(1l, 2l, 45l));
+    verify(traceSink, times(3)).receive(any(Trace.class));
```

Comment:
@adriancole 
I did a bit more investigation. GRPC uses SSL by default, good thing it can be disabled by calling  `io.grpc.ManagedChannelBuilder#usePlaintext`, but unfortunately  GrpcTraceSink doesn't provide easy hooks to make it possible. From GrpcTraceSink constructor

```
  public GrpcTraceSink(String apiHost, Credentials credentials) {
    this.managedChannel = ManagedChannelBuilder.forTarget(apiHost).build();
    this.traceService = TraceServiceGrpc.newBlockingStub(managedChannel)
        .withCallCredentials(MoreCallCredentials.from(credentials));
  }
```

On top of it Wiremock/MockWebServer has to support http/2, probably with good amount of reflection it is doable, but feels dirty.